pipeline/ingest/histology.py

```diff
@@ -136,13 +136,18 @@ def _load_histology_probe(self, key, session, egmap, probe, probepath):
         # probe CCF regions
         names = hist.ont.name
         valid = [isinstance(n, (str,)) for n in names]
-        goodn = np.where(np.array(valid))[0]
 
-        electrodes = (ephys.ProbeInsertion.proj() * lab.ElectrodeConfig.Electrode.proj()
-                      & egmap[probe]).fetch(order_by='electrode asc')
+        probe_electrodes = (lab.ProbeType.Electrode & (ephys.ProbeInsertion & egmap[probe])).fetch(
```

Comment:
meant to mention for the similar query in ephys, can you comment these as a sanity check since technically things should be aligned already?